src/Compiler/FSComp.txt

```diff
@@ -652,6 +652,8 @@ tcExpressionWithIfRequiresParenthesis,"This list or array expression includes an
 808,tcLookupMayNotBeUsedHere,"This lookup cannot be used here"
 809,tcPropertyIsStatic,"Property '%s' is static"
 810,tcPropertyCannotBeSet1,"Property '%s' cannot be set"
+810,tcInitOnlyPropertyCannotBeSet1,"Init-only property '%s' cannot be set outside the initialization code. See https://aka.ms/fsharp-assigning-values-to-properties-at-initialization"
+810,tcGetterForInitOnlyPropertyCannotBeCalled1,"Cannot call '%s' - a setter for init-only property, please use object initialization instead. See https://aka.ms/fsharp-assigning-values-to-properties-at-initialization"
```

Comment:
```suggestion
810,tcSetterForInitOnlyPropertyCannotBeCalled1,"Cannot call '%s' - a setter for init-only property, please use object initialization instead. See https://aka.ms/fsharp-assigning-values-to-properties-at-initialization"
```

Reply:
Thanks, i will need to fix one locally though, since it needs to autogenerate translations.